datafusion/core/src/physical_optimizer/dist_enforcement.rs

```diff
@@ -16,10 +16,20 @@
 // under the License.
 
 //! EnforceDistribution optimizer rule inspects the physical plan with respect
-//! to distribution requirements and adds [RepartitionExec]s to satisfy them
-//! when necessary.
+//! to distribution requirements and adds [`RepartitionExec`]s to satisfy them
```

Comment:
very minor comment is that calling this file `datafusion/core/src/physical_optimizer/enforce_distribution.rs` would be more consistent with the `EnforceDistribtion` rule it contains -- but I realize this is not a change you made in this PR